chain/chain/src/chain.rs

```diff
@@ -3297,6 +3297,12 @@ impl Chain {
         let receipt_ids = outcome.outcome.receipt_ids.clone();
         let mut results = vec![outcome];
         for receipt_id in &receipt_ids {
+            if let Some(receipt) = self.store.get_receipt(&receipt_id)? {
+                let is_refund = receipt.predecessor_id.is_system();
```

Comment:
please add a comment here that we don't include the refund receipts on purpose.

(and please also add the comment to the function itself)